docs/resources/aws_config_delivery_channel.md

```diff
@@ -0,0 +1,77 @@
+---
+title: About the aws_config_delivery_channel Resource
+---
+
+# aws_config_delivery_channel
+
+Use the `aws_config_delivery_channel` InSpec audit resource to test properties of your aws config service.
```

Comment:
Capitalize AWS and Config.  That helps make it clearer that we're talking about the service offered by AWS known as Config.  Also, we need to clarify what this InSpec resource specifically is used for.... maybe:

The AWS Config service can monitor and record changes to your AWS resource configurations.  
Use the `aws_config_delivery_channel` InSpec audit resource to examine how the AWS Config service delivers those change notifications.